apps/cli-daemon/src/app/utils/extract -value-object.ts

```diff
@@ -0,0 +1,49 @@
+import { JsonObject, JsonValue } from '@angular-devkit/core/src/json/utils';
+
+interface JsonObjectNested {
```

Comment:
but why not from the Angular repo?
https://github.com/angular/angular-cli/blob/2aee44174712d3df8d0ffb8434945fbac1c644e4/packages/angular_devkit/core/src/json/utils.ts#L16

Reply:
yes.. no needs for the JsonObjectNested only JsonObject